lib/PasswordStrength/readme.md

```diff
@@ -0,0 +1,27 @@
+# Password Strength
+
+## Introduction
+
+It is POC for password strength component. It requies additional styling, 
+tests and configuration in order to be completed.
+
+## Usage
+
+Include plugin via 
+`<PasswordStrength dataSource={DATA_STRING}></PasswordStrength>`
```

Comment:
Shouldn't this be `password={DATA_STRING}`?

Reply:
Yes, I forgot to update attribute name in the .md. Nice catch.

Reply:
Looks like this still needs to be fixed.